src/pcgnslib.c

```diff
@@ -2165,3 +2165,129 @@ int cgp_array_multi_read_data(int fn, int *A, const cgsize_t *rmin,
 
   return CG_ERROR;
 }
+
+/*===== PointList Functions =============================*/
+
+/**
+ * \ingroup PointListData
+ *
+ * \brief Write index array to PointList in parallel.
+ *
+ * \details Must use functions in @ref AccessingANode to point to a PointSet to read from
+ *
+ * \param[in] file_number \FILE_fn
+ * \param[in] start Index of first point in local array (base 1)
+ * \param[in] end Index of last point in local array (base 1)
```

Comment:
we are using rmin, rmax instead of start and end, also it is the range in the file, not a local array.

Reply:
Ok. For local array, I meant that it's not the entire range of the array in the file (though I see how my phrasing is confusing).

Reply:
I've gone ahead and changed it to "range in the file" since that's the consistent usage from the doxygen branch.